packages/integration-tests/cmr.js

```diff
@@ -163,10 +204,14 @@ async function generateAndStoreCmrXml(granule, collection, bucket) {
 
   const granuleFiles = granule.files.map((f) => f.filename);
 
-  xmlObject.Granule.OnlineAccessURLs.OnlineAccessURL = granuleFiles.map((f) => ({
-    URL: f,
-    URLDescription: 'File to download'
-  }));
+  if (additionalUrls) {
+    xmlObject.Granule.OnlineAccessURLs = additionalUrls.map((url) => ({
+      OnlineAccessURL: {
+        URL: url,
+        URLDescription: 'File to download'
+      }
+    }));
+  }
```

Comment:
I think so too... I think it was working because we blew all the URLs away. Not sure. I did some tests and it seemed to be wrong. I think changing this is what messed up my tests though.